src/main/scala/gnieh/diffson/pointer.scala

```diff
@@ -61,14 +61,10 @@ object / {
 }
 
 private case object Root extends Pointer {
-  override def toString =
-    "/"
+  override def toString = ""
 }
 
 private final case class Path(prefix: Pointer, elem: String) extends Pointer {
   override def toString =
-    prefix match {
-      case Root => f"/${elem.replace("~", "~0").replace("/", "~1")}"
-      case _    => f"${prefix}${elem.replace("~", "~0").replace("/", "~1")}"
-    }
+    s"${prefix}/${elem.replace("~", "~0").replace("/", "~1")}"
```

Comment:
the correct fix would be to change line 72 to be 
```scala
case _    => f"${prefix}/${elem.replace("~", "~0").replace("/", "~1")}"
```

Which it always should have been if I didn’t forget the `/`